src/procevent.c

```diff
@@ -861,7 +861,7 @@ static int procevent_init(void) /* {{{ */
     return (-1);
   }
 
-  if (processlist_head == NULL) {
+  if (ignorelist == NULL) {
```

Comment:
I still don't understand why I keep seeing this change, since it's already part the repo... weird.